### PR TITLE
tidy up documentation around documentation building now that it's dockerized

### PR DESCRIPTION
**Pull request by roastario**

### CODE REVIEW

File: docs/source/building-the-docs.rst [hunk 1]
```diff
@@ -7,77 +7,10 @@ is pre-generated, as well as code documentation, and this can be done automatica
 Requirements
 ------------
 
-In order to build the documentation you will need a development environment set up as described under :doc:`building-corda`.
```
Comment (anthonykeenan) [inline docs/source/building-the-docs.rst]: I think you still need this line, I mean, you need Java and stuff still don't you?

**Review by anthonykeenan — CHANGES_REQUESTED** [review]
Other than the one thing it's fine

**Comment by josecoll** [comment]
Seems like we forgot to merge this for Corda 4.
@fenryka ^^^

**Comment by shamsasari** [comment]
What's the status of this PR?

**Pull request closed without merging**